drivers/interrupt_controller/intc_ite_it8xxx2.c

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright (c) 2020 ITE Corporation. All Rights Reserved
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <kernel.h>
+#include <arch/cpu.h>
+#include <init.h>
+#include <sys/printk.h>
+#include "intc_ite_it8xxx2.h"
+#include <sw_isr_table.h>
+
+static volatile uint8_t *const reg_staus[] = {
+	&ISR0,  &ISR1,  &ISR2,  &ISR3,
+	&ISR4,  &ISR5,  &ISR6,  &ISR7,
+	&ISR8,  &ISR9,  &ISR10, &ISR11,
+	&ISR12, &ISR13, &ISR14, &ISR15,
+	&ISR16, &ISR17, &ISR18, &ISR19,
+	&ISR20
+};
+
+static volatile uint8_t *const reg_enable[] = {
```

Comment:
Please define these arrays to have size CONFIG_NUM_IRQS

Reply:
resolved, the length should be MAX_ISR_REG_NUM.